src/arguments/api_test.py

```diff
@@ -37,26 +38,56 @@ def get_request(cls):
             request_url += endpoint
 
         print("Trying ...\u26A1")
+        return {
+            "request_url" : request_url,
+            "request_headers" : request_headers,
+        }
 
-        # Make GET request and store the response in response_data.json
-        try:
-            response = requests.get(request_url, headers=request_headers)
-            print(f"Reponse Status Code: {response.status_code}")
-            response_data = json.loads(response.content)
-            parsed_json = json.dumps(response_data, indent=4)
-            output_json = highlight(parsed_json, lexers.JsonLexer(),
-                                    formatters.TerminalFormatter())
-            print(output_json)
-
-            store_data = input('Store response data? (Y/N): ')
-            if(store_data == 'Y' or store_data == 'y'):
+    #saves the json response into a file
+    @classmethod
+    def save_response_data(cls,response_data):
+        store_data = input('Store response data? (Y/N): ')
+        if(store_data == 'Y' or store_data == 'y'):
                 with open('response_data.json', 'w') as jsonFile:
                     json.dump(response_data, jsonFile, indent=4)
                 print("Response data stored in response_data.json")
+    # formats the response data and prints it in json on console
+    @classmethod
+    def print_response_json(cls,response):
+        print(f"Reponse Status Code: {response.status_code}")
+        response_data = json.loads(response.content)
+        parsed_json = json.dumps(response_data, indent=4)
+        output_json = highlight(parsed_json, lexers.JsonLexer(),
+                                    formatters.TerminalFormatter())
+        print(output_json)
```

Comment:
Why we are printing and returning the response JSON. It could be either 1 right?
We can print the JSON when we call this method.

Reply:
yeah if we are not returning it, we need to call ```response_data = json.loads(response.content)``` two times in GET, DELETE methods, so what do you suggest?

Reply:
Either return or print. Do one way. You said if you do not return then you need to call this method twice. Then please remove print statement